modules/sandboxed-containers-deleting-kataconfig-crd-cli.adoc

```diff
@@ -0,0 +1,41 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-deleting-kataconfig-CRD-cli_{context}"]
+= Deleting the `KataConfig` custom resource definition using the CLI
+
+The `KataConfig` custom resource definition lets you define the `KataConfig` CR. Delete the `KataConfig` custom resource definition from your cluster.
```

Comment:
The Red Hat supplementary style guide seems to permit `custom resource definition (CRD)` if you wanted to spell out at first use, then use acronym.

Reply:
Done.